### DISCUSSION TITLE
AdvancedMarker with SVG-pin, change anchor point

### DISCUSSION THREAD

**Discussion by jukkahuuskonen** [JustFly1984/react-google-maps-api#3348] [Q&A] [answered]
@Fabioni
Hi,
Do you know how to change the anchorpoint of the svg-pin in advance marker. Our SVGs seem to be anchored in middle of the SVG.
![image](https://github.com/JustFly1984/react-google-maps-api/assets/42405685/598563c6-5d62-4cbd-b41c-db657797ec6f)
instead of middle of the bottom like this (transformed with CSS: ```transform: translateY(-50%);```
![image](https://github.com/JustFly1984/react-google-maps-api/assets/42405685/d0319605-5413-47f5-9720-05e7b75bfaea)

That transform works otherwise just fine, but, if there are markers very close to each other, the one that is just above the other blocks the  lower one from being clickable. For example in this screenshot:
![image](https://github.com/JustFly1984/react-google-maps-api/assets/42405685/46e97d8c-8dc2-42b9-94be-e7316be0c37f)
the middle marker cannot be clicked at all and the lowest marker can be clicked only by clicking very near to the lowest point of the marker.

If I go with the standard marker, it is centered like with the CSS-transform but everyone of these markers can be clicked without any problems:
![image](https://github.com/JustFly1984/react-google-maps-api/assets/42405685/b7472cdc-caa3-4437-a1c1-4549115729d7)
![image](https://github.com/JustFly1984/react-google-maps-api/assets/42405685/ce1cc39b-e609-4d34-8614-c3df2702bd0b)

**Comment by Fabioni** [comment]
Hey. If you are using my implementation. I added a attribute anchorAbove that does exactly this. So you should remove that or set it to false.

**Comment by jukkahuuskonen** [comment, marked as answer]
> anchorAbove

Actually it seems that ```anchorAbove={false/undefined}``` moves the svg down halfway. Setting it to ```true``` seemed to fix the problem. Thanks anyways, without your answer it would have taken me probably quite a bit longer time to figure it out :)
